docs/www/kubernetes-connectivity.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Configuring Kubernetes for connectivity
```

Comment:
what is `for connectivity` it should just be `Configuring Redpanda Operator connectivity`